qemu/tests/block_hotplug_with_max_map_count.py

```diff
@@ -0,0 +1,45 @@
+from avocado.utils import process
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    Block hotplug wit vm.max_map_count test
+    1) Boot the VM with a small max_map_count
+    2) Hotplug 15 block devices
+    3) Login and check the VM status
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    process.system("sysctl vm.max_map_count=3072", shell=True)
+    timeout = params.get_numeric("login_timeout", 240)
+    img_list = params.get("images").split()[3:]
+
+    for img in img_list:
```

Comment:
for idx in range(extra_image_number):
  img="img"+idx
....
you may define the var name you feel good.

Reply:
updated !